website/docs/usage/projects.md

```diff
@@ -227,6 +228,37 @@ pipelines.
 | `workflows`     | A dictionary of workflow names, mapped to a list of command names, to execute in order. Workflows can be run with the [`project run`](/api/cli#project-run) command.                                                                                                                                                                                                                                                                                                                                         |
 | `commands`      | A list of named commands. A command can define an optional help message (shown in the CLI when the user adds `--help`) and the `script`, a list of commands to run. The `deps` and `outputs` let you define the created file the command depends on and produces, respectively. This lets spaCy determine whether a command needs to be re-run because its dependencies or outputs changed. Commands can be run as part of a workflow, or separately with the [`project run`](/api/cli#project-run) command. |
 | `spacy_version` | Optional spaCy version range like `>=3.0.0,<3.1.0` that the project is compatible with. If it's loaded with an incompatible version, an error is raised when the project is loaded.                                                                                                                                                                                                                                                                                                                          |
+### Comet {#comet} <IntegrationLogo name="comet" width={175} height="auto" align="right" />
+
+[Comet.ML](https://www.comet.ml/), like spaCy itself, is out of this
+world. Comet enables teams to track, explain, and optimize spaCy experiments
```

Comment:
As a Wikipedia adminstrator would say - this is not quite [NPOV](https://en.wikipedia.org/wiki/Wikipedia:Neutral_point_of_view) ;-) Let's just remove the first sentence here.

Reply:
It was just making a reference to their space-oriented naming. I'll remove.

Reply:
Done!